tests/e2e/tests/releng_utils.py

```diff
@@ -93,6 +93,20 @@ def fetch_current_fx_product_details(base_url=_product_details_url):
     return releng_products
 
 
+def get_version_info_for_alias(alias, base_url=_product_details_url):
+    """Using up-to-date information drawn from product details service, find the current version of
+    a Firefox product.
+
+    releng_to_bouncer_alias_dict lists the available aliases.
+
+    :param alias: The go-bouncer alias to get the version of. For example, 'firefox-latest'
+    :param base_url: The url of the service that hosts the product details.
+    :return String: The version number of Firefox
+    """
+    products = get_product_mappings()
```

Comment:
You're not passing `base_url` here, so the argument is not being used, and will always default to the value set in `get_product_mappings`.